src/editors/sharedComponents/Checker/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
This component is only found in the problem editor so it should go in the AnswerWidget folder

Reply:
why? It could be used by other components this way, and there is nothing really specific to the answerwidget in there.